packages/java/parser-jvm-core/src/main/java/dev/hilla/parser/models/PropertyInfoReflectionModel.java

```diff
@@ -0,0 +1,55 @@
+package dev.hilla.parser.models;
+
+import java.util.Optional;
+
+import com.fasterxml.jackson.databind.introspect.BeanPropertyDefinition;
+
+final class PropertyInfoReflectionModel extends PropertyInfoModel {
+    private final BeanPropertyDefinition origin;
+
+    PropertyInfoReflectionModel(BeanPropertyDefinition origin,
+            ClassInfoModel owner) {
+        super(owner);
+        this.origin = origin;
+    }
+
+    @Override
+    public BeanPropertyDefinition get() {
+        return origin;
+    }
+
+    @Override
+    public boolean hasGetter() {
+        return origin.hasGetter();
+    }
+
+    @Override
+    public boolean isTransient() {
```

Comment:
We should no longer care about transient stuff.

Reply:
Unfortunately, it's not true. By default, Jackson doesn't handle transient stuff; it just parses transient fields as they are regular fields which is not suitable for us.

Reply:
That's true, but as we discussed some time ago, `transient` exists for on-disk object serialization and does not concern data transfer. Jackson correctly ships those fields. In my opinion we should do the same and don't hide transient fields. I know this is a breaking change, but that's the good moment to make those.